### ISSUE TITLE
Add "All" label to Gmail app is failing (possibly 7.1 migration issue)

### ISSUE THREAD

**Issue by maxelo** [BranchMetrics/sesame_issues#32]
I've been trying to add the "All " Gmail label to sesame shortcuts but all I got is an error message
hope you can fix it
love the app

Screenshot of the problem

https://i.imgur.com/46D3MqE.jpg

https://i.imgur.com/a1oVUys.jpg

![gmail all shortcut 1 bkn](https://user-images.githubusercontent.com/1003307/55121366-f8de7f80-50d8-11e9-9cb3-3ea1a8eeee91.jpg)
![gmail all shortcut 2 bkn](https://user-images.githubusercontent.com/1003307/55121368-f9771600-50d8-11e9-8d2e-e0ac34866880.jpg)

**Comment by philwall2** [comment]
Bummer. I can reproduce the bug. Thanks for reporting.

We'll take a look. FWIW, this might be a growing problem. They method Sesame is using to create those shortcuts is a bit older. Apps have started moving to the newer "7.1 Shortcut" method. It's possible we can find a way to get that working.

**Comment by philwall2** [comment]
NOTE: this also seems to be happening w/ PowerAmp

**Comment by maxelo** [comment]
any news on this bug?

**Comment by philwall2** [comment]
Yes. We investigated and found the problem is indeed that Sesame doesn't have access to launch the 7.1 shortcuts. We need the launcher to give us access. 

We built a fix for this and deployed it in our latest launcher data integration (about a year ago).

Are you on Nova Launcher by chance? They have yet to update to the latest Sesame data.

**Comment by maxelo** [comment]
any news on this bug?
